apps/console/src/app/(protected)/organization-settings/billing-usage/pricing/page.tsx

```diff
@@ -1,12 +1,15 @@
 import React from 'react'
 import { PageHeading } from '@repo/ui/page-heading'
-import PricingPage from '@/components/pages/protected/organization/billing/pricing-table'
+import StripePricingTable from '@/components/pages/protected/organization/billing/pricing-table'
 
 const Page: React.FC = () => {
   return (
     <>
     <PageHeading heading="Pricing Plans" eyebrow="Billing" />
-    <PricingPage />
+    <StripePricingTable 
+      pricingTableId="prctbl_1QKO15Bvxky1R7SvxYRBAjs7"
```

Comment:
these variables aren't used and shouldn't be passed in

Reply:
if i remove them then the table doesn't function?